playground/backend/internal/utils/preparators_utils.go

```diff
@@ -18,24 +18,34 @@ package utils
 import (
 	pb "beam.apache.org/playground/backend/internal/api/v1"
 	"beam.apache.org/playground/backend/internal/preparators"
+	"beam.apache.org/playground/backend/internal/validators"
 	"fmt"
 	"regexp"
+	"sync"
 )
 
 // GetPreparators returns slice of preparators.Preparator according to sdk
-func GetPreparators(sdk pb.Sdk, filepath string) (*[]preparators.Preparator, error) {
-	var prep *[]preparators.Preparator
+func GetPreparators(sdk pb.Sdk, filepath string, valResults *sync.Map) (*[]preparators.Preparator, error) {
```

Comment:
Should we also change this from `GetPreparators` to `GetPreparers`?

Reply:
Done